lib/beaker/host/windows/pkg.rb

```diff
@@ -25,15 +22,6 @@ def install_package(name, cmdline_args = '')
       cygwin = "setup-x86.exe"
     end
 
-    if not check_for_command(cygwin)
```

Comment:
Good point @Iristyle. I think that this points to a guarantee that Windows platforms should have a check for cygwin and install it during initialization. I 'll submit another commit to this PR with those changes and correlating tests.